packages/create-project/commands/init/templates/preact/src/App.jsx

```diff
@@ -7,9 +7,13 @@ export default class App extends Component {
   };
 
   render() {
+    const { name } = this.state;
     return (
       <div class="App">
-        <h1>Welcome to {this.state.name}</h1>
+        <h1>
```

Comment:
Why did this have to be changed?

Reply:
> Why did this have to be changed?

The eslint errors that were being fixed are listed in the commit message

Reply:
Interesting.

Reply:
The "expression on a new line" change here looks a little redundant, but I guess there are other cases where it would help. It may be that eslint-config-prettier soon turns this off or something, but we can always adjust it back when/if we use prettier in this repo :-)